spec/factories/orders.rb

```diff
@@ -0,0 +1,27 @@
+FactoryBot.modify do
+  factory :order do
+    trait :with_line_item do
+      transient do
+        variant { FactoryGirl.create(:variant) }
+      end
+
+      after(:create) do |order, evaluator|
+        create(:line_item, order: order, variant: evaluator.variant)
+      end
+    end
```

Comment:
do we really need this new trait? Spree itself has the `order_with_line_items` factory already. Other than that, this could be done by simply creating two of the three entities and letting FactoryBot create the third one. I'm just concerned about having too many purpose-specific factories that need to be maintained over time. I believe that is already the case in OFN.